lib/utils/errors.go

```diff
@@ -75,3 +75,19 @@ func IsConnectionRefused(err error) bool {
 func IsExpiredCredentialError(err error) bool {
 	return IsHandshakeFailedError(err) || IsCertExpiredError(err) || trace.IsBadParameter(err) || trace.IsTrustError(err)
 }
+
+// IsUntrustedCertErr checks if an error is an untrusted cert error.
+func IsUntrustedCertErr(err error) bool {
+	if err == nil {
+		return false
+	}
+	errMsg := err.Error()
+	return strings.Contains(errMsg, "x509: certificate is valid for") ||
+		strings.Contains(errMsg, "certificate is not trusted")
+}
+
+const (
+	// SelfSignedCertsMsg is a helper message to point users towards helpful documentation.
+	SelfSignedCertsMsg = "Your proxy certificate is not trusted or expired. " +
+		"Please update the certificate or follow this guide for self-signed certs: https://goteleport.com/docs/setup/admin/self-signed-certs/"
+)
```

Comment:
I wasn't really too sure where to put this, but since it applies to errors I left it here. I prefer a constant instead of copy/paste the string in each place, and the code wrapping errors doesn't get all cluttered this way too.